docs/content/en/docs/Examples/cloud_config.md

```diff
@@ -0,0 +1,77 @@
+
+---
+title: "Cloud config examples"
+linkTitle: "Cloud config examples"
+weight: 3
+date: 2017-01-05
+description: >
+  Useful copy-paste cloud-config settings
+---
+
+You can find here examples on how to tweak a system via cloud-config various aspects of an Elemental-toolkit derivative.
+
+The examples are meant to be placed as yaml files under `/oem` ore either `/usr/local/cloud-config`. They can be also given as input cloud-config while calling `elemental install`.
+
+## Networking
+
+### Additional nick
+
+Set static IP with wicked to an additional NIC:
+
+```yaml
+name: "Default network configuration"
+stages:
+   boot:
+     - commands:
+       - wicked ifup eth1
+     - name: "Setup network"
+       files:
+       - path: /etc/sysconfig/network/ifcfg-eth1
+         content: |
+            BOOTPROTO='static'
+            IPADDR='192.168.1.2/24'
+         permissions: 0600
+         owner: 0
+         group: 0
+```
+
+### Static IP
+
+Set static IP with wicked to an additional NIC:
```

Comment:
Not sure if we want to use eth0 here to differentiate from the "Additional nick" section above... 

```suggestion
Set static IP with wicked:
```